### PR TITLE
feat: add backbeat ingestion templates

### PR DESCRIPTION
**Pull request by giacomoguiulfo**
This adds backbeat ingestion to the Zenko stack.

### CODE REVIEW

File: charts/backbeat/templates/ingestion/consumer_deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      labels:
+        app: {{ template "backbeat.name" . }}-ingestion
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ template "backbeat.fullname" . }}-consumer
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/usr/src/app/docker-entrypoint.sh"]
+          args: ["npm", "run", "ingestion_consumer"]
```
Comment (philipyoo) [inline charts/backbeat/templates/ingestion/consumer_deployment.yaml]: `mongo_queue_processor`

File: charts/backbeat/templates/ingestion/consumer_deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          env:
+            - name: REMOTE_MANAGEMENT_DISABLE
+              value: "{{- if .Values.orbit.enabled }}0{{- else }}1{{- end }}"
+            - name: ZOOKEEPER_AUTO_CREATE_NAMESPACE
```
Comment (philipyoo) [inline charts/backbeat/templates/ingestion/consumer_deployment.yaml]: don't think we need zookeeper for the consumer. @jonathan-gramain

File: charts/backbeat/templates/ingestion/consumer_deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+              value: "1"
+            - name: ZOOKEEPER_CONNECTION_STRING
+              value: "{{- printf "%s-zenko-quorum:2181" .Release.Name | trunc 63 | trimSuffix "-" -}}"
+            - name: KAFKA_HOSTS
+              value: "{{- printf "%s-zenko-queue:9092" .Release.Name | trunc 63 | trimSuffix "-" -}}"
+            - name: LOG_LEVEL
+              value: {{ .Values.logging.level }}
+            - name: MONGODB_HOSTS
```
Comment (philipyoo) [inline charts/backbeat/templates/ingestion/consumer_deployment.yaml]: also add `MONGODB_RS`, `MONGODB_DATABASE`

There's also a `readPreference: primary`, but maybe this is an optional?

File: charts/backbeat/templates/ingestion/consumer_deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+spec:
+  replicas: {{ .Values.ingestion.consumer.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "backbeat.name" . }}-ingestion
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ template "backbeat.fullname" . }}-consumer
```
Comment (jonathan-gramain) [inline charts/backbeat/templates/ingestion/consumer_deployment.yaml]: container name can be simply `ingestion-consumer` (as per @ssalaues suggestion on GC)

File: charts/backbeat/templates/ingestion/producer_deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+spec:
+  replicas: {{ .Values.ingestion.producer.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "backbeat.name" . }}-ingestion
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ template "backbeat.fullname" . }}-producer
```
Comment (jonathan-gramain) [inline charts/backbeat/templates/ingestion/producer_deployment.yaml]: same here, `ingestion-producer`

File: charts/backbeat/templates/ingestion/consumer_deployment.yaml (hunk 2 above)
Comment (jonathan-gramain) [reply charts/backbeat/templates/ingestion/consumer_deployment.yaml]: note sure, in doubt we can leave it

**Pull request merged**